channels/in.m3u

```diff
@@ -546,8 +556,8 @@ http://rtmp.santhoratv.zecast.net/santhoratv/santhoratv/playlist.m3u8
 http://santhoratv.zecast.net/santhoratv/santhoratv/index.m3u8
 #EXTINF:-1 tvg-id="SardariTV.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/6Q844bs.jpg" group-title="",Sardari TV (1080p)
 http://158.69.124.9:1935/sardaritv/sardaritv/playlist.m3u8
-#EXTINF:-1 tvg-id="SathiyamTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="",Sathiyam TV (1080p) [Not 24/7]
-https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UC2ziCMHFPWkFHjocUMXT__Q/live
+#EXTINF:-1 tvg-id="SathiyamTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/c/c9/SathiyamFinallogo.png/revision/latest/scale-to-width-down/180?cb=20191225174913" group-title="News",Sathiyam TV (1080p)
+https://iptv.dc-team.com/query?url=https://www.youtube.com/watch?v=zt5J7ZQMZ-A
```

Comment:
Could not get the link, Streamlink couldn't read https://www.youtube.com/watch?v=zt5J7ZQMZ-A

Direct from Youtube:
Private Video
Sign in if you've been granted access to this video

Reply:
Reverted the URL back to older one. Older URL works. Thanks for notifying me!!!